client/header/activity-panel/panels/inbox.js

```diff
@@ -33,6 +33,18 @@ class InboxPanel extends Component {
 		this.props.updateCurrentUserData( userDataFields );
 	}
 
+	handleActionClick( event, note_id, action_id ) {
+		const { triggerNoteAction } = this.props;
+		const href = event.target.href || '';
+
+		if ( 'http' === href.substr( 0, 4 ).toLowerCase() ) {
```

Comment:
It seems like all my links, including internal, include the full protocol and domain (e.g., `http://mylocalwp.test/wp-admin/...`.

Maybe we should compare this to see if it doesn't start with `http` and also doesn't start with `wcSettings.adminUrl`?

Reply:
@joshuatf Do we want to open an order edit or other WC is the current tab or a new one? 

I was using the internal/external division of internal to WooCommerce Admin. I've updated the PR so it works with that division but can easily change to anywhere in the WP dashboard being internal.

Reply:
Not opposed to either, but I would lean towards anything in the dashboard opening in the same tab, regardless of whether its a wc-admin link or not.

Reply:
I agree with @joshuatf, from a user perspective I think there should be no division between WooCommerce Admin pages and other WooCommerce pages, so we should treat all of them the same.

But I don't have a strong opinion in either way so the current solution would also work for me.